src/components/cards/AppInfoCard.tsx

```diff
@@ -170,16 +176,18 @@ const AppInfoCard: React.FC<ComponentProps> = ({
           lastAllocation['Allocation Amount'] ?? '0',
         )
 
-        if (allocationAmount < allowance) {
+        if (
+          allocationAmount < allowance ||
+          isApplicationUpdatedLessThanOneMinuteAgo()
+        ) {
           setIsProgressBarLoading(false)
-          setIsProgressBarVisible(false)
           return
         }
 
         const usedDatacap = allocationAmount - allowance
 
         const progressPercentage = (usedDatacap / allocationAmount) * 100
-
+        setIsProgressBarVisible(true)
         setProgress(progressPercentage)
         setIsProgressBarLoading(false)
```

Comment:
I agree with what you've suggested on the call, we can remove the `isProgressBarLoading` and only leave `isProgressBarVisible`.